docs/config.js

```diff
@@ -0,0 +1,29 @@
+// Valid languages to server-side render in production
+const LANGUAGES = ['en', 'zh', 'pt'];
```

Comment:
```suggestion
const LANGUAGES = ['en'];
```

https://github.com/mui/material-ui/pull/34820 was merged, we no longer support non-en languages.